File: fixtures/python/classes.py
from simple import greet

class User:
    def __init__(self, user_id: str, name: str):
        self.id = user_id
        self.name = name

    def get_display_name(self) -> str:
        return self.name.upper()

    def greet_user(self) -> str:
        return greet(self.name)

File: fixtures/typescript/classes.ts
import { greet } from './simple';

export interface Identifiable {
  id: string;
}

export class User implements Identifiable {
  constructor(public id: string, public name: string) {}

  getDisplayName(): string {
    return this.name.toUpperCase();
  }

  greetUser(): string {
    return greet(this.name);
  }
}
